FileIO/Legacy/MeshIO.h

```diff
@@ -57,7 +57,8 @@ class MeshIO : public Writer
 
 private:
 	void writeElements(std::vector<MeshLib::Element*> const& ele_vec, std::ostream &out) const;
-	MeshLib::Element* readElement(const std::string& line, const std::vector<MeshLib::Node*> &nodes);
+	std::size_t readMaterialID(std::istream & in) const;
+	MeshLib::Element* readElement(std::istream& line, const std::vector<MeshLib::Node*> &nodes) const;
```

Comment:
documentation for both methods is missing ... the name is pretty obvious so i don't know if something is necessary or not ...